code/modules/organs/organ_external.dm

```diff
@@ -1150,10 +1150,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 		var/emote_scream = pick("screams in pain and", "lets out a sharp cry and", "cries out and")
 		owner.emote("me", 1, "[owner.feels_pain() ? emote_scream : ""] drops what they were holding in their [hand_name]!")
 	if(is_malfunctioning())
-		// owner.u_equip(c_hand, 1)
-		owner.emote("me", 1, "drops what they were holding, their [hand_name] malfunctioning!")
-		spark(src, 5, FALSE)
-		owner.drop_item(c_hand)
+		return (is_robotic() && (status & (ORGAN_MALFUNCTIONING) || (brute_dam + burn_dam > min_broken_damage/2 && prob(brute_dam + burn_dam))))
```

Comment:
???

Reply:
I forgot to unfuck stuff from previous PRs and am paying the price